File: backend/ControleFinanceiro.Domain/Entities/Ganho.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ControleFinanceiro.Domain.Entities
{
    public sealed class Ganho
    {
        public long Id { get; set; }
        public string Descricao { get; set; }
        public long CategoriaId { get; set; }
        public Categoria Categoria { get; set; }
        public decimal Valor { get; set; }
        public int Dia { get; set; }
        public long MesId { get; set; }
        public Mes Mes { get; set; }
        public int Ano { get; set; }
        public string UsuarioId { get; set; }
        public Usuario Usuario { get; set; }
    }
}

File: frontend/src/app/models/Ganho.ts
import { Categoria } from "./Categoria";
import { Mes } from "./Mes";

export class Ganho {
  id: number;
  descricao: string;
  categoriaId: number;
  categoria: Categoria;
  valor: number;
  dia: number;
  mesId: number;
  mes: Mes;
  ano: number;
  usuarioId: number;
}
